pages/manual/manual-gettingstarted-build.md

```diff
@@ -87,13 +87,28 @@ Inviwo cannot be compiled with Visual Studio 2022 17.6.x due to a compiler [regr
     ```
 
 2. Clone the Inviwo repository. In the `base` directory run:
-    git clone https://github.com/inviwo/inviwo
+   ```batch
+   git clone https://github.com/inviwo/inviwo
+   ```
 
-3. Clone the vcpkg repository In the `base` directory run:
+3. Clone the vcpkg repository. In the `base` directory run:
+    ```batch
     git clone https://github.com/microsoft/vcpkg
+    ```
+    followed by bootstrapping vcpkg. Optionally, check out the commit listed as "baseline" in inviwo/vcpkg.json to utilize our vcpkg cache.
+    ```batch
+    cd vcpkg
+
+    REM adjust commit hash to match "baseline" in inviwo/vcpkg.json, for example f0f8117
+    git checkout <commithash>
+    
```

Comment:
Put this in a section about caching. 

might do vcpkg/bootstrap.bat instead, otherwise there is a missing cd .. for the next step